test-backend/test-endpoints.js

```diff
@@ -73,27 +72,49 @@ test.serial('after unsuccesfull login it returns 401', async t => {
 
   const res = await request(myServer)
       .post('/api/login')
-      .send({ email: 'a@a.hu', password: 'unsuccesfull login' });
+      .send({ email: 'a@a.hu', password: 'a' });
 
-  t.is(res.status, 401);
+  t.is(res.status, 200);
+  t.true(typeof res.body === 'string');
 
   queryDbStub.reset();
 });
 
-test.serial('after succesfull login it returns 200 status and an object', async t => {
-  t.plan(2);
+test.serial('after register it returns 200 status and an object with the sent email', async t => {
+  t.plan(3);
+
+  const queryDbStub = sinon.stub();
+  queryDbStub.onCall(0).callsArgWithAsync(1, null, null);
+  queryDbStub.onCall(1).callsArgWithAsync(1, null, { email: 'a@a.hu' });
+
+  const myServer = server(queryDbStub);
+
+  const res = await request(myServer)
+      .post('/api/register')
+      .send({ email: 'a@a.hu', password: 'a' });
+
+  t.true(typeof res === 'object');
+  t.true(res.body.email === 'a@a.hu');
+  t.is(res.status, 200);
+
+  queryDbStub.reset();
+});
+
+test.serial('after unsuccesfull register (user exists) it returns 200 status and an object with null', async t => {
+  t.plan(3);
 
   const queryDbStub = sinon.stub();
   queryDbStub.callsArgWithAsync(1, null, { user_id: 1, email: 'a@a.hu', password: 'a' });
 
   const myServer = server(queryDbStub);
 
   const res = await request(myServer)
-      .post('/api/login')
+      .post('/api/register')
       .send({ email: 'a@a.hu', password: 'a' });
 
+  t.true(typeof res === 'object');
+  t.true(!res.body.email);
```

Comment:
`t.equal(res.body.email, null)` would be nicer and more strict. other options: `t.false`, or `t.notOk`